content/root/how-ably-works.textile

```diff
@@ -39,7 +39,7 @@ Ably supports two forms of authentication.
 
 h4. Basic Authentication
 
-Basic authentication is the simplest form of authentication using an app's private API key to connect to Ably. Private API keys and their capabilities (permissions) are managed within an "app's dashboard":http://support.ably.io/support/solutions/articles/3000030053-how-do-i-access-my-app-dashboard. In most cases, we do not recommend that basic authentication is used as it requires that you share your private API key with the client that is connecting to Ably. Our recommendation is to only use basic authentication on your trusted servers when communicating with Ably over TLS.
```

Comment:
Well it's worded wrong, what I meant was it's our recommendation to only use basic auth on your trusted servers, which inadvertently only works over TLS anyway